prov/util/src/util_main.c

```diff
@@ -169,16 +169,10 @@ int util_getinfo(const struct util_prov *util_prov, uint32_t version,
 		return -FI_EINVAL;
 	}
 
-	ret = ofi_check_info(util_prov, version, hints);
+	ret = ofi_check_info_ex(util_prov, version, hints, info);
```

Comment:
The list is handled only [there](https://github.com/ofiwg/libfabric/pull/3065/files#diff-31c0c1d26004d067441bf1a03b5b9d48R176). Seems it's incomplete updates.